### DISCUSSION TITLE
why can't foobar 'hide to tray icon' using this theme?

### DISCUSSION THREAD

**Discussion by CuiYongen** [TT-ReBORN/Georgia-ReBORN#69] [Q&A] [answered]
i found 2 places for this setting:
<img width="550" alt="image" src="https://user-images.githubusercontent.com/12591929/164560315-b22ba843-b0d1-49dc-a2e8-28681523ef5b.png">
and another in 'advanced' (i searched by keyword 'icon')
<img width="549" alt="image" src="https://user-images.githubusercontent.com/12591929/164560374-ca163b0d-3ce6-44da-b0ba-f608dadee889.png">
but still failed(close means exits and not hide to tray). what should i do?

**Comment by TT-ReBORN** [comment]
It's in Display > Columns UI > Notification area tab > Minimise to icon.

**Reply by CuiYongen** [reply]
> It's in Display > Columns UI > Notification area tab > Minimise to icon.

<img width="368" alt="image" src="https://user-images.githubusercontent.com/12591929/164652331-f1e69f2b-8bdf-44bb-8c07-3da112e4c95b.png">

still doesn't work, when i click close button of foobar window, it exits but not hide to tray

**Reply by silverwings15** [reply]
It only MINIMIZES to tray icon, of course closing it would exit the player

**Reply by silverwings15** [reply]
you can probably replace the function of the exit button from closing to minimizing instead to get the desired behavior, but you'd have to wait for TT to explain how

**Reply by CuiYongen** [reply]
> you can probably replace the function of the exit button from closing to minimizing instead to get the desired behavior, but you'd have to wait for TT to explain how

well in default ui when i selceted 'close hides' it can hide to tray by close button, i just want to keep this feature, yet you're right :)

**Reply by CuiYongen** [reply]
i tried to close it using middle mouse button in alt+tab interface, and it successfully hide to tray
so it may be better to adjust the function of the exit button as @silverwings15 mentioned :)

**Comment by TT-ReBORN** [comment, marked as answer]
No, I won't change the close button behavior, it should always close and exit foobar, it would also confuse other users.

If you want the same behavior, it is possible with the minimize button. Go to Display＞Main Window＞Minimize on close: Enable.
This must be also enabled: Display > Columns UI > Notification area tab > Minimise to icon.

-TT

**Reply by silverwings15** [reply]
oh i was suggesting you tell him how to replace the function in his own setup (though i don't know how complicated that is, with all the resizing code and different view modes). obviously changing the default is insanity and i'm not advocating for that 😂

**Reply by CuiYongen** [reply]
> No, I won't change the close button behavior, it should always close and exit foobar, it would also confuse other users.
> 
> If you want the same behavior, it is possible with the minimize button. Go to Display＞Main Window＞Minimize on close: Enable. This must be also enabled: Display > Columns UI > Notification area tab > Minimise to icon.
> 
> -TT

it makes sence.
yet i'm using 'esc key function' to simulate the function without click the button all the time
<img width="231" alt="image" src="https://user-images.githubusercontent.com/12591929/164684007-e1543ecd-3f89-4b88-a09f-e5706c1efdc1.png">
thanks for the explaination :)

**Reply by TT-ReBORN** [reply]
@silverwings15, yea no worries you were right =). But I think this effect can be only achieved with foobar's internals not with Javascript itself, in this case foo_ui_hacks does the job and manipulates the window behavior. So this is the only way to do that.
